drivers/backup/cloud_cred_config.go

```diff
@@ -92,7 +93,9 @@ func GetConfigObj() (*Config, error) {
 	var config Config
 	_, err := os.Getwd()
 	// Read JSON file into a variable
-	testConfigPath := "../drivers/backup/cloud_config.json"
+	cloudConfig := os.Getenv("CLOUD_CRED_CONFIG")
+	testConfigPath := fmt.Sprintf("%s/%s%s", "/mnt", cloudConfig, ".json")
```

Comment:
No need to append `/mnt` let the user provide the absolute path of the file, assuming it is mounted on the correct path of the container/pod

Reply:
Discussed with mithun , i am removing it is not required